tests/string_builder_strings/string_builder_strings.vv

```diff
@@ -0,0 +1,16 @@
+module main
+
+import strings
+
+fn main() {
+	mut buf := strings.new_builder(0)
+	mut test := 'mytest'
+	buf.write_string(test)
+	buf.cap
+	buf.ensure_cap(10)
+	buf.len
+	buf.str()
+	unsafe {
+		buf.free()
```

Comment:
Shouldn't you also set `buf.offset = 0` and `buf.len = 0` in the `unsafe` block?

Reply:
yes you're right